src/Domain/Setting/SettingFactory.ts

```diff
@@ -0,0 +1,70 @@
+import dayjs = require('dayjs')
+import { inject, injectable } from 'inversify'
+import TYPES from '../../Bootstrap/Types'
+import { User } from '../User/User'
+import { Setting } from './Setting'
+import { SettingProps } from './SettingProps'
+import { v4 as uuidv4 } from 'uuid'
+import { CrypterInterface } from '../Encryption/CrypterInterface'
+
+@injectable()
+export class SettingFactory {
```

Comment:
missing `SettingFactoryInterface`

Reply:
Not necessary (YAGNI). E.g. [[1]](https://martinfowler.com/bliki/InterfaceImplementationPair.html), [[2]](https://stackoverflow.com/questions/3036749/is-it-the-best-practice-to-extract-an-interface-for-every-class).

An interface with only one implementation brings no value and is actually more annoying than helpful. It can always be extracted if a need for another implementation arises.